docs/scaling-hive.md

```diff
@@ -0,0 +1,47 @@
+# Scaling Hive
+
+This document describes the main places users are likely to see scalability issues with Hive.
+
+## Hive uses CRDs!
+
+Most importantly, be aware that Hive uses CRDs to store its state. The amount of data that Hive can store is limited by the cluster's ability to store CRs. In OpenShift, this boils down to "how much data can we stuff into etcd before the cluster itself becomes unstable?" Best to not chance it. We recommend keeping the total number of Hive CRs in a Hive cluster in the thousands. Anything approaching 10,000 is the danger zone. In practice this means that a single Hive cluster can manage about 1000 clusters.
```

Comment:
Suggest "should manage no more than 1000 clusters."

Reply:
done